doc/index.rst

```diff
@@ -1,4 +1,4 @@
-.. RSMTool documentation master file, created by
+.. RSMTool documentation main file, created by
```

Comment:
We don't really need to change it here since that's coming from a Sphinx template.

Reply:
My bad! Reverted this change.